File: Level 1/최대공약수와 최소공배수/최대공약수와 최소공배수.py
def gcdlcm(a, b):
    answer = []
    divider = 2
    gcd = 1
    while a >= divider and b >= divider:
        if a % divider == 0 and b % divider == 0:
            a //= divider
            b //= divider
            gcd *= divider
        else:
            divider += 1
    answer.append(gcd)
    answer.append(gcd * a * b)
    return answer

# 아래는 테스트로 출력해 보기 위한 코드입니다.
print(gcdlcm(3,12))

File: Level 1/최대공약수와 최소공배수/최대공약수와 최소공배수.cpp
#include<vector>
#include<iostream>
using namespace std;
vector<int> gcdlcm(int a, int b)
{
	vector<int> answer;
	int gcd = 1;
	int divider = 2;
	while (a >= divider && b >= divider) {
		if (a % divider == 0 && b % divider == 0) {
			a /= divider;
			b /= divider;
			gcd *= divider;
		}
		else {
			divider++;
		}
	}
	answer.push_back(gcd);
	answer.push_back(gcd * a * b);
	return answer;
}

int main()
{
	int a = 3, b = 12;
	vector<int> testAnswer = gcdlcm(a, b);

	cout << testAnswer[0] << " " << testAnswer[1];
}